modules/account-lib/src/coin/hbar/transaction.ts

```diff
@@ -1,10 +1,10 @@
 import { BaseCoin as CoinConfig } from '@bitgo/statics/dist/src/base';
-import { hash } from '@stablelib/sha384';
 import BigNumber from 'bignumber.js';
-import { Writer } from 'protobufjs';
-import * as nacl from 'tweetnacl';
+import { Transaction as SDKTransaction } from '@hashgraph/sdk';
+import { TransactionBody } from '@hashgraph/sdk/lib/generated/TransactionBody_pb';
+import { SignatureMap, SignaturePair } from '@hashgraph/sdk/lib/generated/BasicTypes_pb';
```

Comment:
For what we saw they are not generated every time since they are included in the SDK, probably with a version upgrade for that SDK they could be generated again. But what we saw is that the built version of the SDK includes those generated files.